features/autorisatie/bewoning/raadpleeg-bewoning-met-periode/autorisatie.feature

```diff
@@ -0,0 +1,183 @@
+      Gegeven de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | afnemerID | gemeenteCode |
+      | 000008    | 0800         |
+      Als bewoningen wordt gezocht met de volgende parameters
+      | naam                             | waarde             |
+      | type                             | BewoningMetPeriode |
+      | datumVan                         | 2022-01-01         |
+      | datumTot                         | 2023-01-01         |
+      | adresseerbaarObjectIdentificatie | 0518010000000002   |
+      Dan heeft de response de volgende gegevens
+      | naam     | waarde                                                                                 |
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3                            |
+      | title    | U bent niet geautoriseerd voor deze vraag.                                             |
+      | status   | 403                                                                                    |
+      | detail   | Je mag alleen bewoning van adresseerbare objecten binnen de eigen gemeente raadplegen. |
+      | code     | unauthorized                                                                           |
+      | instance | /haalcentraal/api/bewoning/bewoningen                                                  |
+
+
+  Regel: Een gemeente als afnemer is geautoriseerd voor het vragen van bewoning van een adresseerbaar object dat niet bestaat of nog nooit bewoond is
+
+    Scenario: Het adresseerbaar object wordt niet gevonden, waardoor de gemeente van het adresseerbaarbaar object niet gelijk of ongelijk kan zijn aan de afnemer
+      Gegeven de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | afnemerID | gemeenteCode |
+      | 000008    | 0800         |
+      Als bewoningen wordt gezocht met de volgende parameters
+      | naam                             | waarde             |
+      | type                             | BewoningMetPeriode |
+      | datumVan                         | 2010-01-01         |
+      | datumTot                         | 2010-08-17         |
+      | adresseerbaarObjectIdentificatie | 1234010000123456   |
+      Dan heeft de response 0 bewoningen
+    
+
+  Regel: Een gemeente als afnemer is geautoriseerd voor het vragen van bewoning van een object dat na gemeentelijke herindeling is komen te liggen in bevragende gemeente
+
+    Abstract Scenario: Adres is na gemeentelijke herindeling in vragende gemeente komen te liggen en <omschrijving>
+      Gegeven de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | afnemerID | gemeenteCode |
+      | 000008    | 0800         |
+      En adres 'A3' heeft de volgende gegevens
+      | gemeentecode (92.10) | identificatiecode verblijfplaats (11.80) |
+      | 0530                 | 0530010000000003                         |
+      En adres 'A3' is op '2023-05-26' infrastructureel gewijzigd met de volgende gegevens
```

Comment:
deze stap doet denk ik hetzelfde als:
En adres 'A4' heeft de volgende gegevens
      | gemeentecode (92.10) | identificatiecode verblijfplaats (11.80) |
      | 0800                 | 0530010000000003                         |

Of wordt er nog wel iets gedaan met deze datums?